### DISCUSSION TITLE
Command every crewmember at once

### DISCUSSION THREAD

**Discussion by Crystaxu** [FakeFishGames/Barotrauma#7351] [Ideas and suggestions]
its annoying to give every bot a command one by one so it would be cool to like shift + mouse 1 on a command to command everyone
or maybe selecting multiple crewmembers via a checkbox of some sort?

**Comment by ghost** [comment]
Commanding an entire crew to the same order would be very situational, and it may break the captain's order buffs.

**Reply by Crystaxu** [reply]
Yes, thats why i also said "or maybe selecting multiple crewmembers via a checkbox of some sort?"

**Comment by MaaiG** [comment]
I think we’re talking about different orders being given to the team. Now, you have to pick a team member every time and give them an order through a long sequence of menus.
I’d prefer to see a chart or schedule of all crew members and mark the checkbox of orders all at once (On one menu screen for all crew members). Or drag icon Orders for configure their sequence (priority).
The current system does not ergonomically, IMHO.

**Comment by MateusJK15** [comment]
Something like this![images.jpeg](https://user-images.githubusercontent.com/86162808/143965508-92514944-a7f3-4e47-af23-5c2a051649da.jpeg)

**Reply by MaaiG** [reply]
![Image 001_16-37](https://user-images.githubusercontent.com/93348499/144711584-0917fd4c-aa94-4c0f-af09-77a6233defdf.png)
